src/audio/PlayerSubscription.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/dot-notation */
+import { VoiceConnection } from '../VoiceConnection';
+import { AudioPlayer } from './AudioPlayer';
+
+/**
+ * Represents a subscription of a voice connection to an audio player, allowing
+ * the audio player to play audio on the voice connection.
+ */
+export class PlayerSubscription {
+	/**
+	 * The voice connection of this subscription
+	 */
+	public readonly connection: VoiceConnection;
+
+	/**
+	 * The audio player of this subscription
+	 */
+	public readonly player: AudioPlayer;
+
+	public constructor(connection: VoiceConnection, player: AudioPlayer) {
+		this.connection = connection;
+		this.player = player;
+	}
```

Comment:
This could be done however I don't know how to document those props if they're declared in the constructor directly.
```ts
public constructor(public readonly connection: VoiceConnection, public readonly player: AudioPlayer) {}
```

Reply:
I think we'll skip this one and come back to it later on :+1: